recipes/traja/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 Justin Shenk
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
```

Comment:
(This doesn't prevent this from being merged but so you know for future releases.)

This file shouldn't needed as, according to this line, there should be a copy of the MIT licence included with every distribution of the package. It should be automatically included in the release tarballs on PyPI if you edit your `setup.cfg` to include:

```
[metadata]
license_file = LICENSE
```

Reply:
That's just the information I was looking for, thank you!